doc/configuration-list-evaluation.md

```diff
@@ -0,0 +1,16 @@
+## Configuration list with evaluation evidence
+
+This document contains an iterative table providing details on the certification process of TRUE Connector since its initial release. The table includes information on each version's release date, submodule version, and corresponding details about evaluation evidence that influenced changes in that particular version.
+
+| TRUE Connector version  | Release Date   | Submodules version                                                      | Source code | Evaluation evidence| 
+|:-----------------------:|:--------------:|:-----------------------------------------------------------------------:|:----------:|:-------------------|
+| v1.0.7                  | 2023-02-19     | ECC (v1.14.8), DataApp (v0.3.8), UCApp (v1.7.9), PIP (v1.0.0)           |[Link](https://github.com/Engineering-Research-and-Development/true-connector/releases/tag/v1.0.7) |Teams meetings, Email conversations |
```

Comment:
Should we put here for this version column encoding, AuditLogs in DB?

Reply:
In my opinion, there's no need to explicitly write what has been done since we didn't write anything for all others, and yes - all details about the release can be found in the provided link.